README.md

```diff
@@ -2,6 +2,33 @@
 
 A migration tool does migrations for Konflux pipelines.
 
+## Run tests
+
+```bash
+python3 -m venv venv
+source ./venv/bin/activate
+python3 -m pip install -r requirements-test.txt
+tox
+```
+
+## Local test by running the migration tool
+
+```bash
+source ./venv/bin/activate
+python3 -m pip install -e .
```

Comment:
IMO this should also include `pip install -r requirements.txt` before the `pip install -e .`, to make sure everyone uses the same versions of dependencies. Or, if we have a dedicated README section for the `venv` setup, then we don't really need to mention anything about the venv here